devops/Makefile

```diff
@@ -127,7 +127,7 @@ print-status:
 
 init-project:
 	@echo "Run initial set-up for the project, including network security\n" 
-	@oc process -f openshift/security-init.yml -p NAMESPACE=$$(oc project --short) | oc apply -f -
```

Comment:
We should probably delete the old `security-init.yml` file too

Reply:
I just used the file mentioned in [here](https://developer.gov.bc.ca/Networkpolicy-Migration-Workshop ) and [here](https://github.com/bcgov/networkpolicy-migration-workshop/blob/main/quickstart.yaml)

Reply:
Deleted